runtime/alloc.c

```diff
@@ -272,3 +272,12 @@ CAMLprim value caml_update_dummy(value dummy, value newval)
   }
   return Val_unit;
 }
+
+CAMLexport value caml_alloc_some(value v)
+{
+  CAMLparam1(v);
+  CAMLlocal1(some);
```

Comment:
The `some` variable does not need to be registered, as no GC can occur while it's live.  So you can shave a few cycles by removing this line 279.

Reply:
Thanks, amended as suggested.